pkg/cmd/get.go

```diff
@@ -146,6 +171,38 @@ func (o *GetOptions) Run() error {
 	objs = append(objs, clusterResources...)
 	objs = append(objs, kubearchiveResources...)
 
+	if o.Output != "" {
+		type List struct {
+			Items      []*unstructured.Unstructured `json:"items"`
+			ApiVersion string                       `json:"apiVersion" yaml:"apiVersion"`
+			Kind       string                       `json:"kind"`
+			Metadata   map[string]any               `json:"metadata"`
+		}
+		list := &List{
+			ApiVersion: "v1",
+			Kind:       "List",
+			Metadata:   map[string]any{},
+		}
+		for _, obj := range objs {
+			list.Items = append(list.Items, &obj)
+		}
```

Comment:
Maybe this works the same way:

```suggestion
		list := &List{
			ApiVersion: "v1",
			Kind:       "List",
			Metadata:   map[string]any{},
			List:              objs,
		}
```

Reply:
I think that @rh-hemartin's suggestion is a good one, but let me propose an extra change.
If the struct is going to be used only inside the `if` (where it is defined) there is also the alternative to not declare the struct (which make sense if it isn't going to be referenced):

```go
	if o.Output != "" {
        list := struct {
            Items      []*unstructured.Unstructured `json:"items"`
			ApiVersion string                       `json:"apiVersion" yaml:"apiVersion"`
			Kind       string                       `json:"kind"`
			Metadata   map[string]any               `json:"metadata"`
        }{
            ApiVersion: "v1",
			Kind:       "List",
			Metadata:   map[string]any{},
			List:       objs,
        }
```